hazelcast/src/main/java/com/hazelcast/multimap/impl/MultiMapService.java

```diff
@@ -338,4 +339,16 @@ public void rollbackTransaction(String transactionId) {
     public void dispatchEvent(EventData event, EntryListener listener) {
         dispatcher.dispatchEvent(event, listener);
     }
+
+    @Override
+    public Map<String, LocalMultiMapStats> getStats() {
+        Map<String, LocalMultiMapStats> mapStats = new HashMap<String, LocalMultiMapStats>();
+        for (int i = 0; i < nodeEngine.getPartitionService().getPartitionCount(); i++) {
```

Comment:
Now checked this again, `statsMap`does not contain statistics about even a proxy object is created for it. Statistics are created after a call to `MultiMapProxySupport#invoke`or `MultiMapEventsDispatcher#incrementEventStats`.
We can call `createStats`method in `createDistributedObject`object to create a statistics entry for related `MultiMap`. WDYT?